src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
I guess maybe my contention is that I do not see the advantage of creating a new data item that effectively controls the same plot attribute.  I recognize that the scheme being proposed will not interfere with normal plotting but I think it would be confusing for the user to have new attributes that had `xlimits` and `ylimits` in the names attached to a different object.  I also see a lot of value in being able to switch on and off `use_for_kml` and not change anything from my previous plots and get Google Earth visualization out of it (now come to think of it, why is there a figure type `kml_figure` and this attribute, don't these imply the same thing?)